<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemSettingsForm</class>
 <widget class="QWidget" name="SystemSettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="moveRightButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>90</y>
     <width>70</width>
     <height>380</height>
    </rect>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="moveLeftButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>70</width>
     <height>380</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>801</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>System Settings Form</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="createEveryDay">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Create new data file every day</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>100</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QLabel" name="dataSpaceLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>621</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Space used: 100mb / 400 mb : 25 %</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="cleanLogs">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Clean Logs</string>
   </property>
  </widget>
  <widget class="QLabel" name="updateLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>621</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Version: Build 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>377</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Check For Update</string>
   </property>
  </widget>
  <zorder>Title</zorder>
  <zorder>closeButton</zorder>
  <zorder>moveRightButton</zorder>
  <zorder>moveLeftButton</zorder>
  <zorder>createEveryDay</zorder>
  <zorder>applyButton</zorder>
  <zorder>dataSpaceLabel</zorder>
  <zorder>cleanLogs</zorder>
  <zorder>updateLabel</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
